schemas/EGA.ArrayExperiment.json

```diff
@@ -0,0 +1,109 @@
+      },
+      "sample_number": {
+        "type": "integer",
+        "title": "Number of samples of the ArrayExperiment",
+        "description": "Number of samples included in the experiment. One sample corresponds to one biological replicate [EFO:0002091] (it could be the genetic content from a single cell, a tissue… from a single individual or from several individuals). Shall not be mistaken for technical replicates [CHEBI:24432] being used several times (see https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=EGA&title=Sample+Representation)."
+      },
+
+      "ArrayExperiment_relationships": {
+        "type": "array",
+        "title": "ArrayExperiment relationships",
+        "description": "Comprises metadata (e.g. Source or Target) of a directional association between two entities. This relationshipos node contains all the possible relationships between metadata objects, both outside of (e.g. an Array Design Format that was submitted to ArrayExpress being linked to their microarray data within EGA) and within (e.g. an ArrayExperiment being linked to a Sample) the EGA. #! Using an empty item list but definying the relatiomship as possible additionalItems we create the correct constraint: anything but a relationship object is rejected, but we can add as many as we want.",
+        "additionalProperties": false,
+        "additionalItems": { "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/relationship_object" },
+        "items": []
+      },
+
+      "ArrayExperiment_attributes": {
+        "type": "array",
+        "title": "ArrayExperiment custom attributes",
+        "description": "Custom attributes of an ArrayExperiment: reusable attributes to encode tag-value pairs (e.g. Tag being 'Targeted loci' and its Value '5:63256183-63258334') with optional units (e.g. 'base pairs'). Its properties are inherited from the common-definitions.json schema. #! Using an empty item list but definying the custom attributes as possible additionalItems we create the correct constraint: anything but a custom attribute is rejected, but we can add as many as we want.",
+        "additionalProperties": false,
+        "additionalItems": { "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/custom_attribute" },
+        "items": []
+      },
+
+      "ArrayExperiment_protocols": {
+        "type": "array",
+        "title": "Protocols related to an ArrayExperiment",
+        "description": "Comprises metadata (e.g. Type of protocol) of a plan specification related to an ArrayExperiment, with sufficient level of detail and quantitative information to communicate it (and thus reproduce it) between investigation agents. #! Using an empty item list but definying the protocols as possible additionalItems we create the correct constraint: anything but a protocol is rejected, but we can add as many as we want.",
```

Comment:
Typo:
Using an empty item list but **definying** the protocols > Using an empty item list but **defining** the protocols